packages/catalog-server/src/lib/firestore/custom-element-converter.ts

```diff
@@ -0,0 +1,32 @@
+/**
+ * @license
+ * Copyright 2021 Google LLC
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+import {
+  DocumentData,
+  FirestoreDataConverter,
+  QueryDocumentSnapshot,
+} from '@google-cloud/firestore';
+
+import {CustomElement} from '@webcomponents/catalog-api/lib/schema.js';
+
+export const customElementConverter: FirestoreDataConverter<CustomElement> = {
```

Comment:
Oh the converters run client side, not in Firebase itself? That makes sense I suppose

Reply:
"client side" meaning the Firestore client running on the server, yes.